.travis.yml

```diff
@@ -12,7 +13,7 @@ before_install:
 
 script:
   - set -e
-  - mvn clean verify -Pcoverage
+  - ./.travis.sh
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
```

Comment:
Should this be invoked even when we're on Java9?

Reply:
Yeah, basically it will have no effect on Java 9, the script will not be able to find coverage reports.